src/core/plugins/core-commands/server/commands/moderator/inventory.ts

```diff
@@ -41,19 +41,38 @@ class InventoryCommands {
 
     @command('getitem', LocaleController.get(LOCALE_KEYS.COMMAND_GET_ITEM, '/getitem'), PERMISSIONS.ADMIN)
     private static async handleGetItem(player: alt.Player, ...args) {
-        const fullItemName = args.join(' ');
+        if (args.length === 0) {
+            return Athena.player.emit.message(player, LocaleController.get(LOCALE_KEYS.ITEM_ARGUMENTS_MISSING));
+        }
+
+        if (args.length === 1) {
+            var amount: number = 1;
+            var fullItemName: string = args[0];
+        } else if (args.length >= 2 && isNaN(args[args.length - 1])) {
+            var amount: number = 1;
```

Comment:
var means you're in global scope somehow, so i'd rather replace it by let / const, depending if you want to change it or not